src/dmd/typesem.d

```diff
@@ -906,7 +908,7 @@ private extern (C++) final class TypeSemanticVisitor : Visitor
                 {
                     if (ubyte m = fparam.type.mod & (MODimmutable | MODconst | MODwild))
                     {
-                        mtype.error(loc, "cannot have %s out parameter of type %s", MODtoChars(m), t.toChars());
+                        mtype.error(loc, "cannot have %s out parameter of type `%s`", MODtoChars(m), t.toChars());
```

Comment:
I think "%s out" should also be surrounded by tick marks:  "cannot have \`%s out\` parameter of type \`%s\`"

Reply:
I opted out of `MODtoChars` results being highlighted for the first approximation. There are thousands more error messages to go.